models/chat/telegram.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, token='', client_id=''):
         if not p.match(token):
             raise Exception('Telegram token is invalid')
 
-        p = re_compile(r"^-*\d{7,10}$")
+        p = re_compile(r"^-?\d{7,13}$")
```

Comment:
Nice. It'd be nice if you added unit tests for this change as there's no unit test for our telegram model